test/support/survey_stub.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Ask.BrokerStub do
+defmodule Ask.Runtime.SurveyStub do
```

Comment:
Do you know what's this module for and how it works? Thanks!

Reply:
This module is for mocking runtime/Survey responses. For what I saw, it's used for simplifying channel's test. 

For exaple: NuntiumChannelTest test `"callback with :prompts"`
```elixir
GenServer.cast(SurveyStub.server_ref, {:expects, 
  fn {:sync_step, %Respondent{id: ^respondent_id}, {:reply, "yes"}, "sms"} ->
    {:reply, ReplyHelper.multiple(["Hello!", "Do you exercise?"])}
  end})
```

With this, you're telling the Stub that when receives a `{:reply, ...^respondent_id}` in `sync_step` it should answer you with a `ReplyHelper.multiple(...)`